scripts/sync_issue_to_odoo.py

```diff
@@ -12,11 +20,14 @@
 username = os.getenv("ODOO_USERNAME")  # Odoo login username
 password = os.getenv("ODOO_PASSWORD")  # API Key is used
 project_name = "OpenSPP"  # Project name in Odoo
+issue_title = os.getenv("TITLE", "")[:256]
+issue_body = os.getenv("BODY", "")[:256]
+github_issue_id = os.getenv("ISSUE", "")[:256]
```

Comment:
I think you should be able to do this:
```suggestion
issue_title = sanitize(os.getenv("TITLE", "")[:256])
issue_body = sanitize(os.getenv("BODY", "")[:256])
github_issue_id = sanitize(os.getenv("ISSUE", "")[:256])
```